### PR TITLE
Update add domain

### PR DESCRIPTION
**Pull request by ninjacatcher**
## Phishing Domain/URL/IP(s):
not necessarily, since domains are only used for phishing

## Impersonated domain
electrum.org/com/is

## Describe the issue
phishing

## Related external source

https://www.virustotal.com/gui/domain/electrum.la/detection
https://talosintelligence.com/reputation_center/lookup?search=electrum.la
https://safeweb.norton.com/report/show?url=electrum.la
https://exchange.xforce.ibmcloud.com/url/electrum.la
https://www.virustotal.com/gui/domain/electrum.la/detection
https://talosintelligence.com/reputation_center/lookup?search=electrum.la
https://safeweb.norton.com/report/show?url=electrum.la
https://exchange.xforce.ibmcloud.com/url/electrum.la

https://check.spamhaus.org/results?query=web3-electrum.com
https://check.spamhaus.org/results?query=v2-electrum.com
https://www.virustotal.com/gui/domain/v2-electrum.com
https://www.virustotal.com/gui/domain/web3-electrum.com

https://transparencyreport.google.com/safe-browsing/search?url=elektrum.icu
https://transparencyreport.google.com/safe-browsing/search?url=elektrum.cfd

### CODE REVIEW

File: add-domain [hunk 1]
```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
 wiflix.xyz
 wiflix.yachts
 coronavirus.zone
+xss.ls
```
Comment (spirillen) [inline add-domain:750]: Unconfirmed

image

File: add-domain [hunk 2]
```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
...
+elektrum.sbs
```
Comment (spirillen) [inline add-domain:751]: not active

image

File: add-domain [hunk 3]
```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
...
+www.electrum.la
+electrum.la
```
Comment (spirillen) [inline add-domain:753]: confirmed

image

File: add-domain [hunk 4]
```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
...
+v2-electrum.com
```
Comment (spirillen) [inline add-domain:754]: confirmed

image

File: add-domain [hunk 5]
```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
...
+elektrum.icu
```
Comment (spirillen) [inline add-domain:755]: unconfirmed

image

File: add-domain [hunk 6]
```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
...
+elektrum.cfd
```
Comment (spirillen) [inline add-domain:756]: Unconfirmed

image

File: add-domain [hunk 7]
```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
...
+web3-electrum.com
```
Comment (spirillen) [inline add-domain:757]: Working, but not convinced it is phishing

image

**Review by spirillen — CHANGES_REQUESTED** [review]
As a bare minimum I'll have to request more information for adding the domains marked with negative comments before they can be added.

This is why screenshots, the field you deleted from issue template, are so damned important + the field of `Where you found the risk ware`,

**Comment by ninjacatcher** [comment]
Please check the detection score. All of them are used for phishing. They are even flagged by Microsoft Defened SmartScreen, Google Safebrowsing and Spamhaus.
[a0e20c3](https://github.com/mitchellkrogza/phishing/pull/530/commits/a0e20c36fc26d7588a6a4aea85b5b607a941da07) its old commit. please ignore this.

File: add-domain (hunk 7 above)
Comment (ninjacatcher) [reply add-domain:784]: Please check the detection score. All of them are used for phishing. They are even flagged by Microsoft Defened SmartScreen, Google Safebrowsing and Spamhaus.

elektrum[.]icu & elektrum[.]cfd has been taken down.

1. **https://check.spamhaus.org/results/?query=www.electrum.la**
2. Microsoft Defender SmartScreen
3. https://radar.cloudflare.com/domains/domain/electrum.la
4. https://www.virustotal.com/gui/domain/electrum.la
5. https://urlfiltering.paloaltonetworks.com/single_cr/?url=electrum.la
6. https://www.quad9.net/result/?url=electrum.la
7. https://exchange.xforce.ibmcloud.com/url/electrum.la
8. https://talosintelligence.com/reputation_center/lookup?search=electrum.la
9. https://safeweb.norton.com/report/show?url=electrum.la
10. https://maltiverse.com/hostname/www.electrum.la
11. https://reports.adguard.com/en/electrum.la/report.html

1. **https://check.spamhaus.org/results?query=web3-electrum.com**
2. **https://check.spamhaus.org/results?query=v2-electrum.com**
3. https://safeweb.norton.com/report/show?url=v2-electrum.com
4. https://www.virustotal.com/gui/domain/v2-electrum.com
5. https://www.virustotal.com/gui/domain/web3-electrum.com
6. https://safeweb.norton.com/report/show?url=web3-electrum.com

msedge_G7RYC04fWq

File: add-domain (hunk 4 above)
Comment (ninjacatcher) [reply add-domain:781]: Please check the detection score. All of them are used for phishing. They are even flagged by Microsoft Defened SmartScreen, Google Safebrowsing and Spamhaus.

elektrum[.]icu & elektrum[.]cfd has been taken down.

1. **https://check.spamhaus.org/results/?query=www.electrum.la**
2. Microsoft Defender SmartScreen
3. https://radar.cloudflare.com/domains/domain/electrum.la
4. https://www.virustotal.com/gui/domain/electrum.la
5. https://urlfiltering.paloaltonetworks.com/single_cr/?url=electrum.la
6. https://www.quad9.net/result/?url=electrum.la
7. https://exchange.xforce.ibmcloud.com/url/electrum.la
8. https://talosintelligence.com/reputation_center/lookup?search=electrum.la
9. https://safeweb.norton.com/report/show?url=electrum.la
10. https://maltiverse.com/hostname/www.electrum.la
11. https://reports.adguard.com/en/electrum.la/report.html

1. **https://check.spamhaus.org/results?query=web3-electrum.com**
2. **https://check.spamhaus.org/results?query=v2-electrum.com**
3. https://safeweb.norton.com/report/show?url=v2-electrum.com
4. https://www.virustotal.com/gui/domain/v2-electrum.com
5. https://www.virustotal.com/gui/domain/web3-electrum.com
6. https://safeweb.norton.com/report/show?url=web3-electrum.com

msedge_G7RYC04fWq

File: add-domain (hunk 2 above)
Comment (spirillen) [reply add-domain:778]:
```
pyfunceble -a -d xss.ls

    ██████╗ ██╗   ██╗███████╗██╗   ██╗███╗   ██╗ ██████╗███████╗██████╗ ██╗     ███████╗
    ██╔══██╗╚██╗ ██╔╝██╔════╝██║   ██║████╗  ██║██╔════╝██╔════╝██╔══██╗██║     ██╔════╝
    ██████╔╝ ╚████╔╝ █████╗  ██║   ██║██╔██╗ ██║██║     █████╗  ██████╔╝██║     █████╗
    ██╔═══╝   ╚██╔╝  ██╔══╝  ██║   ██║██║╚██╗██║██║     ██╔══╝  ██╔══██╗██║     ██╔══╝
    ██║        ██║   ██║     ╚██████╔╝██║ ╚████║╚██████╗███████╗██████╔╝███████╗███████╗
    ╚═╝        ╚═╝   ╚═╝      ╚═════╝ ╚═╝  ╚═══╝ ╚═════╝╚══════╝╚═════╝ ╚══════╝╚══════╝
    


Subject                                                                                              Status      Source     Expiration Date   HTTP Code  Checker      
---------------------------------------------------------------------------------------------------- ----------- ---------- ----------------- ---------- -------------
xss.ls                                                                                               ACTIVE      WHOIS      12-jul-2025       Unknown    AVAILABILITY
```

**Review by spirillen — APPROVED** [review]

**Pull request merged**